TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -211,6 +211,45 @@ function LocateHighestVersionSqlPackageWithSql()
 
 function LocateHighestVersionSqlPackageWithDacMsi()
 {
+    $sqlDataTierFrameworkRegKeyWow = "HKLM:", "SOFTWARE", "Wow6432Node", "Microsoft", "Microsoft SQL Server", "Data-Tier Application Framework" -join [System.IO.Path]::DirectorySeparatorChar
```

Comment:
Ajay, you are working on a utility, right?
If possible, can you pull this registry checks into that utility?